From: Alison Chaiken <alison_chaiken@mentor.com>
Date: Fri, 16 May 2014 09:25:53 +0200
Subject: man: readahead: fix cmdline switch inconsistency between readahead.c
 and docs

Source code has "files-max" and XML has --max-files.

(cherry picked from commit 332bc31992acffc6f32e194c0122e01607bd0e27)
---
 man/systemd-readahead-replay.service.xml | 2 +-
 src/readahead/readahead.c                | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/systemd-readahead-replay.service.xml b/man/systemd-readahead-replay.service.xml
index 67b41f5b..669fe789 100644
--- a/man/systemd-readahead-replay.service.xml
+++ b/man/systemd-readahead-replay.service.xml
@@ -117,7 +117,7 @@
 
                 <variablelist>
                         <varlistentry>
-                                <term><option>--max-files=</option></term>
+                                <term><option>--files-max=</option></term>
 
                                 <listitem><para>Maximum number of
                                 files to read ahead. Only valid
diff --git a/src/readahead/readahead.c b/src/readahead/readahead.c
index d6729ec9..73cf5380 100644
--- a/src/readahead/readahead.c
+++ b/src/readahead/readahead.c
@@ -42,7 +42,7 @@ static int help(void) {
                "Collect read-ahead data on early boot.\n\n"
                "  -h --help                 Show this help\n"
                "     --version              Show package version\n"
-               "     --max-files=INT        Maximum number of files to read ahead\n"
+               "     --files-max=INT        Maximum number of files to read ahead\n"
                "     --file-size-max=BYTES  Maximum size of files to read ahead\n"
                "     --timeout=USEC         Maximum time to spend collecting data\n\n\n",
                program_invocation_short_name);
